kas/base.yml

```diff
@@ -0,0 +1,54 @@
+header:
+    version: 19
+
+distro: nile
+
+defaults:
+    repos:
+        branch: "scarthgap"
+
+local_conf_header:
+    disk-space-monitoring: |
+        BB_DISKMON_DIRS ??= "\
+            STOPTASKS,${TMPDIR},1G,100K \
+            STOPTASKS,${DL_DIR},1G,100K \
+            STOPTASKS,${SSTATE_DIR},1G,100K \
+            STOPTASKS,/tmp,100M,100K \
+            HALT,${TMPDIR},100M,1K \
+            HALT,${DL_DIR},100M,1K \
+            HALT,${SSTATE_DIR},100M,1K \
+            HALT,/tmp,10M,1K"
+
+    ipk-packages: |
+        PACKAGE_CLASSES ?= "package_ipk"
+
+repos:
+    meta-nile:
+        path: "meta-nile"
+        layers:
+            .:
+    bitbake:
+        url: "https://github.com/openembedded/bitbake.git"
+        path: "layers/bitbake"
+        branch: "2.8"
+        layers:
```

Comment:
Can you add a comment about why this is disabled. e.g., `# this repo does not provide any layers` like in [kas userguide](https://kas.readthedocs.io/en/latest/userguide/project-configuration.html#:~:text=%23%20this%20repo%20does%20not%20provide%20any%20layers).